transaction/std/src/transaction_builder.rs

```diff
@@ -57,6 +57,18 @@ pub struct TxOutContext {
     pub shared_secret: RistrettoPublic,
 }
 
+/// Signing data for external library
```

Comment:
I think this requires much documentation to explain why it is here and whats the intended use is.